assets/app/views/directives/osc-key-values.html

```diff
@@ -1,95 +1,175 @@
 <div ng-controller="KeyValuesController" class="labels">
     <div class="form-inline labels-edit" ng-show="editable">
-      <ng-form class="edit-label" name="form" novalidate>
-        <div class='form-group' ng-class="{'has-error': form.key.$error.oscKeyValid}">
-          <input class="form-control"
-                 type="text"
-                 name="key"
-                 ng-attr-placeholder="{{keyTitle}}"
-                 ng-model="key"
-                 autocorrect="off"
-                 autocapitalize="off"
-                 spellcheck="false"
-                 osc-input-validator="key">
-        </div>
-        <div class="form-group" ng-class="{'has-error': form.value.$error.oscValueValid}">
-          <input class="form-control"
-                 type="text"
-                 name="value"
-                 ng-attr-placeholder="{{valueTitle}}"
-                 ng-model="value"
-                 autocorrect="off"
-                 autocapitalize="off"
-                 spellcheck="false"
-                 osc-input-validator="value">
+      <form class="edit-label" name="form" novalidate>
+
+        <div layout row>
+
+          <div
+            flex grow="5"
+            class='form-group'
+            ng-class="{'has-error': form.key.$error.oscKeyValid}"
+            style="margin-right: 10px;">
+            <input class="form-control"
+                   type="text"
+                   name="key"
+                   ng-attr-placeholder="{{keyTitle}}"
+                   ng-model="key"
+                   autocorrect="off"
+                   autocapitalize="off"
+                   spellcheck="false"
+                   osc-input-validator="key">
+          </div>
+
+          <div
+            flex grow="5"
+            class="form-group"
+            ng-class="{'has-error': form.value.$error.oscValueValid}"
+            style="margin-right: 10px;">
+            <input class="form-control"
+                   type="text"
+                   name="value"
+                   ng-attr-placeholder="{{valueTitle}}"
+                   ng-model="value"
+                   autocorrect="off"
+                   autocapitalize="off"
+                   spellcheck="false"
+                   osc-input-validator="value">
+          </div>
+          <div
+            flex layout
+            grow="1" shrink="1" basis="auto"
```

Comment:
I can get it to this, still with flex:
![screen shot 2016-02-15 at 11 27 38 am](https://cloud.githubusercontent.com/assets/280512/13054306/2d48b98e-d3d7-11e5-8063-9700c2553205.png)

Setting `cross-axis: start` on the parent keeps the button from stretching, and so long as it doesn't have a `flex` attr but the inputs do, it works.